src/components/draft/editor/Store.ts

```diff
@@ -0,0 +1,53 @@
+import produce from 'immer';
+import { devtoolsInNonProd } from 'utils/store-utils';
+import create from 'zustand';
+
+interface EditorStorState {
+    draftId: string | null;
+    setDraftId: (newVal: EditorStorState['draftId']) => void;
+
+    currentCatalog: string | null;
+    setCurrentCatalog: (newVal: EditorStorState['currentCatalog']) => void;
```

Comment:
Might put the file list in here? Maybe... would just make setting the default file easier.